### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by melashu**
# In this milestone I have made the following changes.

- Add `species` table.
- Add `owners` table.
- Remove the `species` column from animals.
- Add owner_id and species_id as foreign keys to the animal's table.
- Insert owner's data.
- Insert species data.
- Query data from multiple tables. 

### Here is the screenshot of my query result.

**Screenshot 1**

**Screenshot 2**

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -8,4 +8,20 @@ CREATE TABLE animals (
     neutered BOOLEAN NOT NULL,
     weight_kg DECIMAL NOT NULL,
 );
-ALTER TABLE animals ADD species VARCHAR(20);
\ No newline at end of file
+ALTER TABLE animals ADD species VARCHAR(20);
+
```
Comment (KDlamini) [inline schema.sql:12]: In this milestone, the SQL that modifies the animal's table should REMOVE the column species not ADD it.

File: data.sql [hunk 2]
```diff
@@ -10,4 +10,15 @@ INSERT INTO animals (name,neutered,escape_attempts,date_of_birth,weight_kg) VALU
 INSERT INTO animals (name,neutered,escape_attempts,date_of_birth,weight_kg) VALUES ('Angemon',TRUE,1,'2005-06-12',-45);
 INSERT INTO animals (name,neutered,escape_attempts,date_of_birth,weight_kg) VALUES ('Boarmon',TRUE,7,'2005-06-07',20.4);
 INSERT INTO animals (name,neutered,escape_attempts,date_of_birth,weight_kg) VALUES ('Blossom',TRUE,3,'1998-10-13',17);
-INSERT INTO animals (name,neutered,escape_attempts,date_of_birth,weight_kg) VALUES ('Ditto',TRUE,4,'2022-05-14',22);
\ No newline at end of file
+INSERT INTO animals (name,neutered,escape_attempts,date_of_birth,weight_kg) VALUES ('Ditto',TRUE,4,'2022-05-14',22);
+
+INSERT INTO owners (full_name,age) VALUES ('Sam Smith',34),('Jennifer Orwell',19),('Bob',45),('Melody Pond',77),('Dean Winchester',14),('Jodie Whittake',38);
+
```
Comment (KDlamini) [inline data.sql]: - Kindly make sure that the SQL for adding `Pokemon` and `Digimon` to the `species` table is also present in the `data.sql` file as per project requirements.

image

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @melashu 🙋‍♀️,  

Great work so far! 💯 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞
 
✔️  Correct PR title and description
✔️  Correct files are used for SQL

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (melashu) [reply schema.sql:12]: Hi  @KDlamini, Thank you so much for reminding me.

**Comment by ShoiraTa** [comment]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @melashu , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

### Highlights:
 ✔️ Project is setup correctly
 ✔️ All linters are passing
✔️ Correct gitflow
 ✔️ Professional documentation

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

image

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@shoirata] in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**